new-findings.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
I'm not sure this script should live in the root.  I'd prefer introducing a `scripts/` or `hack/` folder personally.

Reply:
I would add that this script should be referenced in the developer docs, and maybe it should replace the existing script in `DEVELOPING.md`